docs/sources/flow/reference/components/faro.receiver.md

```diff
@@ -0,0 +1,267 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/faro.receiver/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/faro.receiver/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/faro.receiver/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/faro.receiver/
+title: faro.receiver
+---
+
+# faro.receiver
+
+`faro.receiver` accepts web application telemetry data from the [Grafana Faro Web SDK][faro-sdk]
+and forwards it to other components for future processing.
+
+[faro-sdk]: https://github.com/grafana/faro-web-sdk
+
+## Usage
+
+```river
+faro.receiver "LABEL" {
+    output {
+        logs   = [LOGS_RECEIVERS]
```

Comment:
Is this an OTel log receiver (like traces below), or a loki.* component?

*edit*: I see this is explained in the `output` block below, nvm.